docs/installation.rst

```diff
@@ -57,22 +61,10 @@ This will create a ``dist`` directory in ``treeherder-ui`` where concatenated an
 Configuration
 =============
 
-You can either run the treeherder service locally, or use a remote server.
-This setting is specified in this file:
-
+The sample configuration makes the UI load job/push data from the production service API. If you wish to test the UI against stage/dev's service instead, adjust thServiceDomain in this file:
 ``webapp/app/js/config/local.conf.js``
 
-A sample copy of this file is located here:
-
-``webapp/app/js/config/sample.local.conf.js``
-
-Copy the sample file to ``local.conf.js`` and make your custom settings.
-If you want to run the UI using the node web server shipped with it, you need to change this line::
-    
-    window.thServiceDomain = "";
-
-to::
-
-    window.thServiceDomain = "https://treeherder.mozilla.org";
+If you wish to run the full treeherder-service Vagrant project (service + UI), remember to remove local.conf.js or else change thServiceDomain within it to use |same_host|, so the UI will use the local Vagrant service API.
```

Comment:
This |same_host| would need adjusting also if we switch the config, but more interestingly, when I view this file this phrase is linked, but just gives me an identical installation page.